package.json

```diff
@@ -12,8 +12,8 @@
 		"prestart": "run-p setup clean db:dev",
 		"init": "run-p db",
 		"start": "run-p start:main start:worker",
-		"start:main": "tsnd -r tsconfig-paths/register --watch=.env*,db/config.json,**/*.hbs --inspect=0.0.0.0:9229 --respawn --transpile-only -- src/main.ts",
-		"start:worker": "tsnd -r tsconfig-paths/register --watch=.env*,db/config.json,**/*.hbs --inspect=0.0.0.0:9230 --respawn --transpile-only -- src/worker.ts",
+		"start:main": "tsnd -r tsconfig-paths/register --watch=.env*,db/config.json --inspect=0.0.0.0:9229 --respawn --transpile-only -- src/main.ts",
```

Comment:
Just to be sure, we don't need to watch the hbs files here as they required dynamically.